internal/cmd/forwarder-generator/main.go

```diff
@@ -40,18 +40,17 @@ func main() {
 		os.Exit(1)
 	}
 	log.V(1).Info("Args: %v", os.Args)
-
 	var reader func() ([]byte, error)
 	switch *yamlFile {
 	case "-":
-		reader = func() ([]byte, error) { return ioutil.ReadFile(*yamlFile) }
-	case "":
-		reader = func() ([]byte, error) { return []byte{}, nil }
-	default:
 		reader = func() ([]byte, error) {
 			stdin := bufio.NewReader(os.Stdin)
 			return ioutil.ReadAll(stdin)
 		}
+	case "":
```

Comment:
i had the same changes in my syslog-functional-tests PR. but i never get to finish it